spec/factories/offsprings.rb

```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+  factory :offspring do
+    first_name "MyString"
+    last_name "MyString"
+    type ""
```

Comment:
I think this code was actually generated by rails when I created the Model @sergio-ocon